package.json

```diff
@@ -18,19 +18,25 @@
     "fs"
   ],
   "author": "Thomas Bergwinkl <bergi@axolotlfarm.org> (https://www.bergnet.org/people/bergi/card#me)",
+  "contributors": [
+    "Brett Zamir"
+  ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/bergos/file-fetch/issues"
   },
   "homepage": "https://github.com/bergos/file-fetch",
+  "engines": {
```

Comment:
It should be aligned with the Travis config or maybe we don't add it at all. I would prefer the last option. I never checked in detail what effect the `engines` property has. I think Node would throw an error, if the version doesn't match. As the compatible versions are sync with the LTS versions, I see it as the users responsibility to handle problems caused by unsupported Node versions.

Reply:
`engines` only has an impact if the user opts in to the config `engine-strict`, and then will err out only if the package is added as a dependency.

Besides flagging for users whether they may continue using an older (or later) Node version, this field has been used by linting tools to draw attention to whether one mistakenly introduces syntax or features which are outside of the advertised `engines` range. But since the rules of "standard" do not make use of this (despite being potentially available through one of their peer deps, `eslint-plugin-node`, e.g., see https://github.com/mysticatea/eslint-plugin-node/tree/master/docs/rules/no-unsupported-features , and even if you were using this, it defaults to LTS versions anyways), this latter advantage appears immaterial in this case.

I've updated to drop `engines`.